test/server/cards/03_TWI/events/InPursuit.spec.ts

```diff
@@ -0,0 +1,35 @@
+describe('In Pursuit', function() {
+    integration(function(contextRef) {
+        describe('In Pursuit\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['in-pursuit'],
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['green-squadron-awing']
+                    },
+                    player2: {
+                        groundArena: ['pyke-sentinel'],
+                        spaceArena: ['inferno-four#unforgetting']
+                    }
+                });
+            });
+
+            it('should exhaust an enemy unit if you exhaust a friendly unit', function () {
```

Comment:
Can you add tests for when there is no friendly unit to exhaust? I suspect this will still let you exhaust an enemy unit. I also believe this requires the friendly unit to actually go from not exhausted to to exhausted, so we should add a test with only an exhausted friendly unit

Reply:
ok, added a test that verifies if you do not have a unit to exhaust the card does nothing